pkg/accountmanager/iamrole.go

```diff
@@ -0,0 +1,124 @@
+package accountmanager
```

Comment:
what about `childaccountmanager`?
It's longer, but I got a little confused, thinking this package was for managing `Account` struct objects